app/models/intake_form.rb

```diff
@@ -22,7 +22,24 @@ class IntakeForm
                 :sponsor_group
 
   validates :citizenship_status, inclusion: { in: SponsorGroup::CITIZENSHIP_STATUSES }
-  validates :name, :email, :phone, presence: true
+  validates :refugee_connection_type, inclusion: { in: SponsorGroup::REFUGEE_CONNECTION_TYPES }
+  validates :name,
+            :address_line_1,
+            :city,
+            :province,
+            :postal_code,
+            :phone,
+            :email,
+            :citizenship_status,
+            :group_size,
+            :sah_connection,
+            :interpreter_needed,
+            :sufficient_resources,
+            :connected_to_refugee,
+            :refugee_outside_country_of_origin, presence: true
+  validates_format_of :postal_code, with: /\A[ABCEGHJKLMNPRSTVXY]{1}\d{1}[A-Z]{1} *\d{1}[A-Z]{1}\d{1}\z/
+  validates_format_of :email, with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i
+  validates_format_of :phone, with: /\A^[\+0-9\s\-\(\)]+$\z/
```

Comment:
Prefer the new style validations `validates :column, format: value` over `validates_format_of`.